src/OSPSuite.Core/Domain/Mappers/EventAssignmentBuilderToEventAssignmentMapper.cs

```diff
@@ -1,37 +1,61 @@
+using System.Collections.Generic;
+using System.Linq;
 using OSPSuite.Core.Domain.Builder;
 
 namespace OSPSuite.Core.Domain.Mappers
 {
    /// <summary>
-   /// Maps one event assignment from building block into model
+   ///    Maps one event assignment from building block into model
    /// </summary>
-   public interface IEventAssignmentBuilderToEventAssignmentMapper : IBuilderMapper<IEventAssignmentBuilder, IEventAssignment>
+   public interface IEventAssignmentBuilderToEventAssignmentMapper : IBuilderMapper<IEventAssignmentBuilder, IReadOnlyList<IEventAssignment>>
```

Comment:
Instead of using the keyword replacer for this, we need to CREATE new entries so I decided to do it in the mapper which is the place where we register all new entries